src/main/app/forms/models/dateOfBirth.ts

```diff
@@ -16,23 +16,27 @@ export class ValidationErrors {
 }
 
 export default class DateOfBirth implements Serializable<DateOfBirth>, CompletableTask {
+  @IsDefined ({ message: 'Choose option: Yes or Not sure' })
+  known: boolean
 
+  @ValidateIf(o => o.known === true)
   @ValidateNested()
   @IsValidLocalDate({ message: ValidationErrors.DATE_NOT_VALID })
   @isValidYearFormat(4, { message: ValidationErrors.DATE_INVALID_YEAR })
   @MinimumAgeValidator(18, { message: ValidationErrors.DATE_UNDER_18 })
   @MaximumAgeValidator(150, { message: ValidationErrors.DATE_NOT_VALID })
   date: LocalDate
 
-  constructor (date: LocalDate = new LocalDate()) {
+  constructor (known?: boolean, date: LocalDate = new LocalDate()) {
+    this.known = known
     this.date = date
   }
 
   static fromObject (value?: any): DateOfBirth {
     if (!value) {
       return value
     }
-    return new DateOfBirth(LocalDate.fromObject(value.date))
+    return new DateOfBirth(Object.getOwnPropertyDescriptor(value, 'known') !== undefined ? value.known === 'true' : undefined, LocalDate.fromObject(value.date))
```

Comment:
Can we simplify it to be just `value.known === 'true'`. I do not think the longer version does anything for us at the moment due to the way form values are deserialised upon submission.